<launch>
	<arg name="ns" default="human1"/>
	<arg name="map_name" default="laas_adream"/>

	<node name="humanBehaviorModel_node" type="humanBehaviorModel_node" pkg="inhus" ns="$(arg ns)" output="screen"> 
		<param name="map_name" value="$(arg map_name)"/>
	</node>
	<node name="taskPlanner_node" type="taskPlanner_node" pkg="inhus" ns="$(arg ns)" output="screen">
		<param name="map_name" value="$(arg map_name)"/>
	</node>
	<node name="supervisor_node" type="supervisor_node" pkg="inhus" ns="$(arg ns)" output="screen"> 
		<param name="map_name" value="$(arg map_name)"/>
	</node>
	<node name="logManager_node" type="logManager_node" pkg="inhus" ns="$(arg ns)">
		<param name="map_name" value="$(arg map_name)"/>
	</node>
	<node name="lowLevelController_node" type="lowLevelController_node" pkg="inhus" ns="$(arg ns)">
		<param name="map_name" value="$(arg map_name)"/>
	</node>

</launch>
